src/mumble/AudioOutputToken.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+#define MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+
+#include "AudioOutput.h"
+#include "AudioOutputToken.h"
+
+#include <QObject>
```

Comment:
```suggestion
#include <QObject>

#include <cassert>
```
(for the `assert` macro)